test/test_signalfd.rs

```diff
@@ -11,8 +11,7 @@ use nix::unistd;
 fn main() {
     print!("test test_signalfd ... ");
 
-    let mut mask = signal::SigSet::empty();
-    mask.add(signal::SIGUSR1).unwrap();
+    let mask: signal::SigSet = signal::SIGUSR1.into();
```

Comment:
In particular, I'm not really convinced this particular characterbake makes sense:

```rust
pub fn signalfd<'a, T: Into<&'a SigSet>>(fd: RawFd, mask: T, flags: SfdFlags) -> Result<RawFd>
```

In particular, I'm not sure what will actually fulfill the bound `Into<&'a SigSet>` other than `&SigSet` -- making it just a waste of characters.